apps/admin/backend/src/app.ts

```diff
@@ -166,13 +171,81 @@ function buildApi({
       return auth.unprogramCard(constructAuthMachineState(workspace));
     },
 
-    async writeBallotPackageSignatureFile(input: {
-      ballotPackagePath: string;
-    }): Promise<void> {
-      await artifactAuthenticator.writeSignatureFile({
-        type: 'ballot_package',
-        path: input.ballotPackagePath,
+    async saveBallotPackageToUsb(): Promise<Result<void, 'no_usb_drive'>> {
```

Comment:
I'm wondering if we should give errors an explicit type here, even though there's only 1 possible error value at the moment. This feels a bit like a magic string - is this common practice for the `Result` abstraction when there's only 1 possible error value?

Reply:
Good call! A shared type feels much clearer: https://github.com/votingworks/vxsuite/pull/3554/commits/2ec12c1f5a79001e58c9a9b923d85048614b97c1